aviator-cli/rebasing-and-syncing-with-github.md

```diff
@@ -0,0 +1,64 @@
+# Rebasing and Syncing with GitHub
+
+When you work on a stack of branches, the state of the stack can be out of sync
+with the remote or a parent branch. For example, if the `main` branch on the
+remote gets updates by merging other pull-requests, or a new commit is added or
+a commit is amended in one of the parent branches, or some of the parents are
+merged, you want to rebase your branches on top of the new commits.
+
+`av stack sync` is an action that does that for you. Aviator CLI fetches the
+latest commits from the remote, and rebases your branches on top of the new
+commits. There are some variants of the sync operations depending on your needs.
+
+## Normal Sync
+
+Running `av stack sync` without any option does the followings:
+
+* Handle the merged branches
+  * If a branch is merged, its child branches are reparented to the trunk
+    branch.
+  * Those reparented branches are rebased on top of the latest trunk branch.
+  * You are asked to delete those merged branches in the command.
+* Rebase the out-of-sync branches
+  * If a parent is amended or a new commit is added, the child branches are
+    rebased on top of the new commit.
+  * This is done only for the branches whose parent is not trunk.
+* Push the changes to the remote if there's a remote branch.
+
+Note that the sync operation without any option does not rebase the branches to
```

Comment:
This can be a little confusing, we are saying that when there are merged branches that it would rebase on top of the latest trunk and here we are saying that it doesn't do that. Perhaps explain that distinction